packages/server/package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "@sigstore/server",
+  "version": "0.0.0",
+  "private": "true",
+  "description": "Sigstore Server",
```

maybe call out it's only for testing in case we publish this at some point?
```suggestion
  "description": "Sigstore mock server for testing purposes only",
```